translations/strings.yaml

```diff
@@ -671,6 +671,15 @@ subscriptionManagement:
   planValueYearly:
     value: "%1 Yearly"
     comment: The %1 represents a localized currency string.
+  planValueMonthlyPlusTax:
+    value: "%1 Monthly + tax"
+    comment: The %1 represents a localized currency string. This string is only used in US and Canada
```

Comment:
Is the comment accurate about US and Canada? That would be surprising.

Still, I would remove it, because we might want to use this elsewhere in the future.